### PR TITLE
Functionality to track login history of User

### LINKED ISSUE #5652: Implement User login tracking details
## What type(s) of user does this feature affect?
 All

## Description
To implement functionality of MAU (Monthly Active Users) and WAU (Weekly Active Users) by Role as mentioned in issue #5530 , we need to implement functionality to track login history of User.

## How to access the QA site
_Login Details:_  
[Link to QA site](https://casa-qa.herokuapp.com/)  

Login Emails: 
- volunteer1@example.com  view site as a volunteer
- supervisor1@example.com view site as a supervisor
- casa_admin1@example.com view site as an admin
- all_casa_admin1@example.com view site as an all casa admin
  - go to `/all_casa_admins/sign_in`  

password for all users: 12345678  

### Questions? Join Slack!

We highly recommend that you join us in [slack](https://join.slack.com/t/rubyforgood/shared_invite/zt-21pyz2ab8-H6JgQfGGI0Ab6MfNOZRIQA) #casa channel to ask questions quickly. And [discord](https://discord.gg/qJcw2RZH8Q) for office hours (currently Tuesday 5-7pm Pacific), stakeholder news, and upcoming new issues.

### PR DESCRIPTION
**Pull request by sarvaiyanidhi**
### What github issue is this PR for, if any?
Resolves #5652 

### What changed, and why?
In order to create chart for tracking Weekly Active User and Monthly Active Users, we need to track login history of User. This issue implements the functionality which will log login activity of any user in new table login_activities.

### How will this affect user permissions?
- Volunteer permissions: NA
- Supervisor permissions: NA
- Admin permissions: NA

### How is this tested? (please write tests!) 💖💪
Record is logged in login_activities table after user login.

### Screenshots please :)
NA

### Feelings gif (optional)
What gif best describes your feeling working on this issue? https://giphy.com/
How to embed:
`alt text`

### Feedback please? (optional)
We are very interested in your feedback! Please give us some :) https://forms.gle/1D5ACNgTs2u9gSdh9

### CODE REVIEW

File: app/controllers/users/sessions_controller.rb [hunk 1]
```diff
@@ -3,4 +3,9 @@
 class Users::SessionsController < Devise::SessionsController
   include Accessible
   skip_before_action :check_user, only: :destroy
+  after_action :after_login, only: :create
+
+  def after_login
+    LoginActivity.create!(user_id: current_user.id, email: current_user.email, current_sign_in_ip: current_user.current_sign_in_ip, current_sign_in_at: Time.now, user_type: current_user.type)
```
Comment (compwron) [inline app/controllers/users/sessions_controller.rb:9]: maybe we should use https://github.com/ankane/authtrail ? What do you think?

Comment (sarvaiyanidhi) [reply app/controllers/users/sessions_controller.rb:9]: Had looked into this option but thought as we need to just log history details so don't want to add another gem dependency but can update code if that looks better option in place of custom code?

Comment (sarvaiyanidhi) [reply app/controllers/users/sessions_controller.rb:9]: @compwron Have re-implemented functionality using Authtrail gem and raised new PR #5706 for it. Can you please review it whenever possible.. I will close this PR after approval.

**Comment by sarvaiyanidhi** [comment]
Closing this PR in preference to alternate implementation done in #5706

**Pull request closed without merging**